crates/forge/tests/cli/cmd.rs

```diff
@@ -12,7 +12,6 @@ use foundry_test_utils::{
     util::{OutputExt, TestCommand, read_string},
 };
 use semver::Version;
-use similar_asserts::assert_eq;
```

Comment:
How are these changes related to the zksync deps - most seem to be foundry-specific tests.